docs/source/peers/peers.md

```diff
@@ -369,61 +447,79 @@ can be accepted by the network. This is literally what it means to achieve
-therefore inconsistent transaction responses to the application *for the same
-transaction proposal*. It might simply be that the result was generated at
-different times on different peers with ledgers at different states, in which
-case an application can simply request a more up-to-date proposal response. Less
-likely, but much more seriously, results might be different because the chaincode
-is *non-deterministic*. Non-determinism is the enemy of chaincodes
-and ledgers and if it occurs it indicates a serious problem with the proposed
-transaction, as inconsistent results cannot, obviously, be applied to ledgers.
-An individual peer cannot know that their transaction result is
-non-deterministic --- transaction responses must be gathered together for
-comparison before non-determinism can be detected. (Strictly speaking, even this
-is not enough, but we defer this discussion to the transaction section, where
-non-determinism is discussed in detail.)
-
-At the end of phase 1, the application is free to discard inconsistent
-transaction responses if it wishes to do so, effectively terminating the
-transaction workflow early. We'll see later that if an application tries to use
-an inconsistent set of transaction responses to update the ledger, it will be
-rejected.
+-->
+
+Phase 1 is complete for a successful ledger update proposal when the client
+application receives signed responses from the gateway service that satisfies
+the endorsement polic. Note that endorsement peers can return different and
+therefore inconsistent transaction proposal responses. This variation can be
+caused by peers running the transaction at different times with ledgers in
+varying states of being updated, in which case the application can simply
+request another response to the proposal. Less likely, but much more seriously,
+results of a transaction may vary because the invoked chaincode is *non-deterministic*.
+Non-deterministic chaincode is code that produces different results over time
+for the same transaction, indicating a problem with the chaincode's reliability.
+By design, blockchain prohibits inconsistent results from being written to the
+ledger, regardless of the cause.
+
+<!-- (Strictly
+speaking, even this is not enough; this discussion continues in the transaction
+section where non-determinism is discussed in more detail.)
+
+target section or discussion not found -->
+
+An individual peer cannot detect whether its transaction result is
+non-deterministic --- transaction responses must be collected by the gateway
+service for comparison before it can detect non-determinism in chaincode.
+In response to inconsistent transaction responses, caused either by
+nondeterministic chaincode or any other issue, the gateway service terminates
+the transaction workflow and returns a failed proposal message to the client. The
+client can then resubmit the transaction proposal or an amended transaction proposal.
```

Comment:
I realize this content existed previously, but it becomes more confusing with gateway.
I would remove all this discussion of different endorsements and non-determinism. It is too much information and too nuanced for Key Concepts. It gets in the way more than helps, especially now that gateway is in the middle.

Reply:
okay will comment out for a future move to another location.